packages/main/src/types/ToolbarDesign.ts

```diff
@@ -0,0 +1,13 @@
+const ToolbarDesign = {
+	/**
```

Comment:
This and the other types can be enums to be consistent with the rest of the types. Also we can add the @type tag to the jsdocs as well as a short description of what to expect when using them.